File: program/swap-two-numbers-without-using-third-variable/SwapTwoNumbersWithoutUsingThirdVariable.cs
using System;
 
class GFG {
    public static void Main()
    {
        int x = 10;
        int y = 20;
 
        x = x + y;
        y = x - y;
        x = x - y;
        Console.WriteLine("After swapping: x = " + x + ", y = " + y);
    }
}
 

File: program/program/swap-two-numbers-without-using-third-variable/swap_two_numbers_without_using_third_variable.dart
import 'dart:io';

void main() {
  // Input two numbers
  stdout.write('Enter the first number: ');
  int num1 = int.parse(stdin.readLineSync()!);
  stdout.write('Enter the second number: ');
  int num2 = int.parse(stdin.readLineSync()!);

  // Swap the two numbers
  num1 = num1 + num2;
  num2 = num1 - num2;
  num1 = num1 - num2;

  // Output the swapped numbers
  print('After swapping:');
  print('First number = $num1');
  print('Second number = $num2');
}